File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h, w, space, hash, hash2; // h is height  w is width  hash2 for the second pyramid
    
    
    do
    {
        printf("Height: ");
        h =get_int();
        w=h;
    }
    while (0>h || h>23);
    
    for (; h>0; h--)
 {
    for (space=h; space>1; space--)
    {
        printf(" ");
    }
    for (hash=h; hash<w+1; hash++)
    {
        printf("#");
    }
    
    printf("  ");
    for (hash2=h; hash2<w+1; hash2++)
    {
        printf("#");
    }
    printf("\n");
 }
    
}

File: mario.py
import cs50


def main():
    while True:
        print("Height:", end="")
        h = cs50.get_int()
        if h > 0 and h < 23:
         break
    width = h
    for i in range(h):
        print(" " * (h - i - 1), end="")
        print("#" * (i + 2), end="")    # "i + 2": +1 because i starts from 0 and +1 for pyramid
        print("  ", end="")
        print("#" * (i+2))
        print()

if __name__=="__main__":
    main()